docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,162 @@
+.. _acsincome-data:
+ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
+different algorithmic fairness interventions. ACSIncome offers a few improvements,
+such as providing more datapoints (1,664,500 vs. 48,842) and more recent data (2018 vs. 1994).
+Further, the binary labels in the UCI Adult dataset indicate whether an indivdual earned 
+more than $50k US dollars in that year. Ding et al. show that the choice of threshold impacts
+the amount of fairness violation, so they allow users to define any threshold.
+
+Ding et al. compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. filtered the data such that ACSIncome only includes individuals above 16 years old 
+who worked at least 1 hour per week in the past year and had an income of at least $100.
+
+
+.. _acsincome_dataset_description:
+
+Dataset Description
+^^^^^^^^^^^^^^^^^^^
+Ding et al. provide data from 2014-2018 for all 50 states and Puerto Rico.
```

Comment:
I'm undecided on whether it makes sense to explicitly mention that other territories are excluded. What do you think?

Reply:
In my opinion, the documentation should just describe what is included and not necessarily enumerate everything that is excluded, but I'm open

Reply:
I'm thinking from my perspective as someone who wasn't born in the US. I don't think I knew about anything beyond 50 states, let alone territories like Guam, until I moved to the US. From that mistaken viewpoint it's easy to assume you're querying all of the US and describe it as such. In other words, this nuance is potentially lost on people. Part of why we're documenting this is to avoid missing nuances like that. This is the application context that's so crucial. If we train a model on data from 50 states + PR we can't assume it works elsewhere (not even the territories). [In fact, we shouldn't necessarily assume it "works" anywhere unless we validate it in its real world context.]

So perhaps this makes sense as elaborating a little bit on why it's a slippery slope, and the more information the better.

Reply:
Thanks for clarifying. I added a sentence of "Note that Puerto Rico is the only US territory included in this dataset." Let me know if we should provide more detail like explicitly naming the territories that are excluded